app/views/users/registrations/new.html.haml

```diff
@@ -5,19 +5,19 @@
       = devise_error_messages!
       .field
         = f.label :name, class: 'principal-label'
-        = f.text_field :name, autofocus: true, class: 'sing-up-field'
+        = f.text_field :name, autofocus: true, class: 'principal-field'
       .field
         = f.label :email, class: 'principal-label'
-        = f.email_field :email, class: 'sing-up-field'
+        = f.email_field :email, class: 'principal-field'
       .field
         = f.label :password, class: 'principal-label'
-        = f.password_field :password, autocomplete: 'off', placeholder: "MIN. #{@minimum_password_length} CHARACTERS LONG", class: 'sing-up-field'
+        = f.password_field :password, autocomplete: 'off', placeholder: "MIN. #{@minimum_password_length} CHARACTERS LONG", class: 'principal-field'
       .field
         = f.label :password_confirmation, class: 'principal-label'
-        = f.password_field :password_confirmation, autocomplete: 'off', class: 'sing-up-field'
+        = f.password_field :password_confirmation, autocomplete: 'off', class: 'principal-field'
       .field
         = f.label :gender, class: 'principal-label'
-        = f.select :gender, [['Male', 1], ['Female', 2]], {}, { class: 'select-gender' }
+        = f.select :gender, [['Male', 1], ['Female', 2]], {}, class: ['principal-field', 'select-gender']
```

Comment:
You can just write: `class: 'principal-field select-gender'`. It's a little bit cleaner.